pkg/network/encoding/http.go

```diff
@@ -97,13 +99,14 @@ func newHTTPEncoder(payload *network.Connections) *httpEncoder {
 	return encoder
 }
 
-func (e *httpEncoder) GetHTTPAggregationsAndTags(c network.ConnectionStats) (*model.HTTPAggregations, uint64) {
+func (e *httpEncoder) GetHTTPAggregationsAndTags(c network.ConnectionStats) (*model.HTTPAggregations, uint64, map[string]struct{}) {
 	if e == nil {
-		return nil, 0
+		return nil, 0, nil
 	}
 
 	keyTuple := network.HTTPKeyTupleFromConn(c)
```

Comment:
The heuristic used by this function to determine which side of the connection is the client & which is the server  doesn't work on windows, so we need to modify this function to return both possible keys & check for aggregations for both keys: see https://github.com/DataDog/datadog-agent/blob/aeb4e8d7a1a1c73c165bda6401a051c1be4fe3dd/pkg/network/event_common.go#L383-L396 and https://github.com/DataDog/datadog-agent/blob/aeb4e8d7a1a1c73c165bda6401a051c1be4fe3dd/pkg/network/encoding/http.go#L61-L66